bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_WWbb/Template/proc_card.dat

```diff
@@ -0,0 +1,22 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set gauge unitary
+set complex_mass_scheme False
```

Comment:
Hi @ammitra , I think none of these lines is actually necessary and can be safely removed, as indeed it has been done in your reference commit https://github.com/cms-sw/genproductions/commit/39c661dde790c6ace22994a67d56aa7ad2ae586a